cnd/src/http_api/routes.rs

```diff
@@ -60,7 +61,7 @@ pub async fn handle_get_halight_swap(
                 (alpha_ledger_state, beta_ledger_state, finalized_swap)
             }
             _ => {
-                let empty_swap = make_swap_entity(swap_id, vec![]);
+                let empty_swap = siren::Entity::default().with_class_member(http_api::PATH); // FIXME: Is this 'swaps' (http_api::PATH) correct?
```

Comment:
This should not use the path, it has nothing to do with URLs: https://github.com/kevinswiber/siren#class